Resources/Prototypes/DeltaV/Objectives/eviltwin.yml

```diff
@@ -0,0 +1,31 @@
+- type: entity
+  abstract: true
+  parent: BaseTerminatorObjective # mrp terminator real
+  id: BaseEvilTwinObjective
+  components:
+  - type: Objective
+    issuer: self
+
+- type: entity
+  noSpawn: true
+  parent: [BaseEvilTwinObjective, BaseKillObjective]
+  id: EvilTwinKillObjective
+  description: Follow your programming and terminate the target.
```

Comment:
Afaik paradox anomalies are literally just a bluespace thing, not something programmed. Additionally, their objective shouldn't have a "Win" condition, as killing your double is optional

Reply:
yes its copypaste from terminator while i was looking for the old desc then realised it didnt have one :trollface:

Reply:
holy shit im cooking

giving it both kill and help objective so there is literally no way to "win" since no greentext
roundend will show if you made friends with target (alive and they are too) or replaced them (alive and they arent)